src/index_next_region.js

```diff
@@ -2,8 +2,6 @@
 exports.handler = (event, context, callback) => {
   var region = event.regions.shift();
   event.num_of_regions = event.regions.length;
-  event.cloudtrail.queryStringParameters.region = region;
```

Comment:
Why are we removing the region part

Reply:
We are not following the next region step for cloudtrail so I added this line in the "index_health_stack.js" just before making call to cloudtrail lambda

Reply:
Config rules and Cloudtrail were the parallel tasks, wont that be needed by Config rules?